db/seeds.rb

```diff
@@ -47,7 +47,7 @@
 hermione = User.where(username: "hogwartsahistory").first
 hermione.subscriptions.create([
     {subscription_name: "Flourish and Blotts", username: "hermione", password: "wingardiumleviOsa", url: "flourish.com", cost_per_month: 4.99},
-    {subscription_name: "Hogwarts Tuition Portal", username: "granger", password: "password", url: "hogwarts.com", cost_per_month: 1495.95}
+    {subscription_name: "Hogwarts Tuition Portal", username: "granger", password: "password", url: "hogwarts.com", cost_per_month: 105.95}
```

Comment:
wow discounted hogwarts

Reply:
yes I did haha. can't have that huge bar dwarfing all of the other things